--- tmp/8.14.3-decompiled/XPackBuild.original.java
+++ tmp/8.14.3-decompiled/XPackBuild.pached.java
@@ -20,30 +20,7 @@
   static {
     String shortHash, date;
     Path path = getElasticsearchCodebase();
-    if (path.toString().endsWith(".jar")) {
-      try {
-        JarInputStream jar = new JarInputStream(Files.newInputStream(path, new java.nio.file.OpenOption[0]));
-        try {
-          Manifest manifest = jar.getManifest();
-          shortHash = manifest.getMainAttributes().getValue("Change");
-          date = manifest.getMainAttributes().getValue("Build-Date");
-          jar.close();
-        } catch (Throwable throwable) {
-          try {
-            jar.close();
-          } catch (Throwable throwable1) {
-            throwable.addSuppressed(throwable1);
-          } 
-          throw throwable;
-        } 
-      } catch (IOException e) {
-        throw new RuntimeException(e);
-      } 
-    } else {
-      shortHash = "Unknown";
-      date = "Unknown";
-    } 
-    CURRENT = new XPackBuild(shortHash, date);
+    CURRENT = new XPackBuild("Unknown", "Unknown");
   }
   
   @SuppressForbidden(reason = "looks up path of xpack.jar directly")
