--- tmp/8.7.0-decompiled/XPackBuild.original.java
+++ tmp/8.7.0-decompiled/XPackBuild.pached.java
@@ -1,51 +1,24 @@
 package org.elasticsearch.xpack.core;
 
-import java.io.IOException;
 import java.net.URISyntaxException;
 import java.net.URL;
-import java.nio.file.Files;
 import java.nio.file.Path;
-import java.util.jar.JarInputStream;
-import java.util.jar.Manifest;
 import org.elasticsearch.core.PathUtils;
 import org.elasticsearch.core.SuppressForbidden;
 
 public class XPackBuild {
   public static final XPackBuild CURRENT;
-  
+
   private String shortHash;
-  
+
   private String date;
-  
+
   static {
     String shortHash, date;
     Path path = getElasticsearchCodebase();
-    if (path.toString().endsWith(".jar")) {
-      try {
-        JarInputStream jar = new JarInputStream(Files.newInputStream(path, new java.nio.file.OpenOption[0]));
-        try {
-          Manifest manifest = jar.getManifest();
-          shortHash = manifest.getMainAttributes().getValue("Change");
-          date = manifest.getMainAttributes().getValue("Build-Date");
-          jar.close();
-        } catch (Throwable throwable) {
-          try {
-            jar.close();
-          } catch (Throwable throwable1) {
-            throwable.addSuppressed(throwable1);
-          } 
-          throw throwable;
-        } 
-      } catch (IOException e) {
-        throw new RuntimeException(e);
-      } 
-    } else {
-      shortHash = "Unknown";
-      date = "Unknown";
-    } 
-    CURRENT = new XPackBuild(shortHash, date);
+    CURRENT = new XPackBuild("Unknown", "Unknown");
   }
-  
+
   @SuppressForbidden(reason = "looks up path of xpack.jar directly")
   static Path getElasticsearchCodebase() {
     URL url = XPackBuild.class.getProtectionDomain().getCodeSource().getLocation();
@@ -53,18 +26,18 @@
       return PathUtils.get(url.toURI());
     } catch (URISyntaxException bogus) {
       throw new RuntimeException(bogus);
-    } 
+    }
   }
-  
+
   XPackBuild(String shortHash, String date) {
     this.shortHash = shortHash;
     this.date = date;
   }
-  
+
   public String shortHash() {
     return this.shortHash;
   }
-  
+
   public String date() {
     return this.date;
   }
